domains/POAS/events/material-change-confirmed/schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.lpastore/material-change-confirmed.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.lpastore/material-change-confirmed",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "^M(-[A-Z0-9]{4}){3}$"
+    },
+    "actorType": {
+      "type": "string",
+      "description": "The type of actor that made the change",
+      "enum": ["donor", "certificateProvider", "attorney"]
+    },
+    "actorUID": {
+      "type": "string",
+      "description": "The UID of the actor that made the change",
+      "pattern": "^([a-z0-9]{8}-)([a-z0-9]{4}-){3}([a-z0-9]{12})$"
+    }
+  },
```

Comment:
Q for Sirius people: will marking this as a material change trigger some communication we could include the date of in here? Since we want to have "We contacted you on {{.Date}} with guidance about what to do next." displayed, it would be nice to have the sent date